tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -307,3 +309,25 @@ export const getSourceFileList = createSelector(
     return state.sourceCode.sourceFileList;
   }
 );
+
+export const getFocusedSourceFileIndex = createSelector(
```

Comment:
Optional, but wdyt about `getFocusedSourceFileIndex` --> `getFocusedSourceFile(): sourceFileSpec`?

I suspect clients need the actual SourceFileSpec rather than the fileIndex.

Reply:
There is a specific use case where the numeric index is needed: the call site for `fetchSourceFile()` in debugger_effects.ts.

Reply:
Ah I didn't realize that the HTTP API itself is aware of fileIndices.  Let's continue with this, then.